css/_uswds-theme-custom-styles.scss

```diff
@@ -802,7 +806,7 @@ body {
   }
 }
 
-.preview-no_border {
+.preview-no__border {
```

Comment:
Was this class even used? Not seeing any templates using either.

If it's not being used, let's remove the styles.

Reply:
Removed in [68a4160](https://github.com/uswds/uswds-site/pull/1573/commits/68a4160c2289b20ae11c9a89a23e97cb59f3b6a4)